packages/textkit/src/attributedString/advanceWidth.js

```diff
@@ -1,14 +1,16 @@
 import runAdvanceWidth from '../run/advanceWidth';
 
+/**
+ * @typedef {import('../types.js').AttributedString} AttributedString
+ */
+
 /**
  * Returns attributed string advancewidth
  *
- * @param {Object} attributedString attributed string
+ * @param {AttributedString} attributedString attributed string
  * @returns {number} advance width
  */
-const advanceWidth = (attributedString) => {
+export default function advanceWidth(attributedString) {
```

Comment:
Nit. Any reason to switch to functions? I kind of prefer arrow fns. Not important but curious :)

Reply:
Also personal preference (and affects bundle size a bit, but that's not really relevant for react-pdf I guess). Still, regardless of motives, that is out of scope :)